### ISSUE TITLE
Cannot comment on documents with commenting enabled

### ISSUE THREAD

**Issue by jackbuehner** [jackbuehner/cristata-issues#9]
From Colin: The comment button is grayed out

**Comment by jackbuehner** [comment]
The "Save" button in the comments editor is grayed out, but the comment buttons in the ribbon and selection flyout are active.
The create button in the comments sidebar panel is also deactivated.

![image](https://github.com/jackbuehner/cristata-issues/assets/16235094/6a52c437-93ab-4327-b158-ba158d5d1e30)

**Comment by jackbuehner** [comment]
@ColinBrake This is now fixed in the update rolled out a few hours ago.

There was logic in place to prevent creating comments on things that cannot have comments (e.g., photos or YouTube videos). However, that logic was also interfering with the ability to save comments. It now no longer interferes with comment saving. (jackbuehner/cristata-app@a51de73d05fcb2d596f12417bb5466d9fffd4c66)

I also found and fixed (jackbuehner/cristata-app@059901f71af0d52279eac7ce48195bebe1e79b48) an unrelated bug that made it difficult to delete comments that were only on one letter.
